client/src/quic_client.rs

```diff
@@ -108,7 +108,7 @@ impl QuicNewConnection {
 
     // If this function becomes public, it should be changed to
     // not expose details of the specific Quic implementation we're using
-    async fn create_endpoint(config: EndpointConfig, client_socket: UdpSocket) -> Endpoint {
```

Comment:
I think the comment also doesn't apply anymore now that it's not on `QuicClient`

Reply:
Removed